packages/oas/src/extensions.ts

```diff
@@ -228,6 +228,11 @@ export interface Extensions {
      * @example "Custom cURL snippet"
      */
     name?: string;
+    /**
+     * Optional template engine for dynamic code samples
+     * @example "mustache"
+     */
+    templateEngine?: string;
```

Comment:
debating making this an enum / hardcode values. thoughts?

Reply:
Definitely make this an enum.

Reply:
done!